audio/pavucontrol/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                pavucontrol
+version             4.0
+license             GPL-2+
+categories          audio
+maintainers         {mit.edu:quentin @quentinmit} \
+                    openmaintainer
+platforms           darwin
+description         PulseAudio Volume Control
+
+long_description    PulseAudio Volume Control is a GTK+ volume control tool for PulseAudio.
+
+homepage            https://freedesktop.org/software/pulseaudio/pavucontrol/
+master_sites        https://www.freedesktop.org/software/pulseaudio/pavucontrol/
```

Comment:
Might as well just write this once. Set one of them to the other, like you did with the livecheck.url